tests/kernel/common/src/clock.c

```diff
@@ -40,21 +40,23 @@ void test_clock_uptime(void)
 
 	/**TESTPOINT: uptime elapse*/
 	t64 = k_uptime_get();
-	while (k_uptime_get() < (t64 + 5))
+	while (k_uptime_get() < (t64 + 5)) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```

Comment:
I don't think you need the `;` anymore

Reply:
yeah, was aware of that, but was too lazy, busted :(